docs/ci-cd-environment/environment-variables.md

```diff
@@ -340,6 +340,67 @@ name of the Pull Request.
 
 Example values: `Update Readme.md`
 
+### Environment Variables injected into after_pipeline jobs
+
+The following environment variables are injected only in
+[after pipeline jobs][after-pipeline-ref]{target="_blank"}.
+
+#### SEMAPHORE\_PIPELINE\_RESULT
+
+The value of the `SEMAPHORE_PIPELINE_RESULT` contains the result of the pipeline.
+
+Values: `failed`, `passed`, `canceled`, `stopped`.
+
+#### SEMAPHORE\_PIPELINE\_RESULT\_REASON
+
+The value of the `SEMAPHORE_PIPELINE_RESULT_REASON` contains the reason for
+the pipeline result.
+
+Values: `test`, `malformed`, `stuck`, `internal`, `user`, `strategy`, `timeout`.
+
+#### SEMAPHORE\_PIPELINE\_TOTAL\_DURATION
+
+The value of the `SEMAPHORE_PIPELINE_TOTAL_DURATION` contains the duration of
+the pipeline including queuing time.
+
+The value is expressed in seconds.
+
+#### SEMAPHORE\_PIPELINE\_INIT\_DURATION
+
+The value of the `SEMAPHORE_PIPELINE_INIT_DURATION` contains the duration of
+the pipeline initialization.
+
+The value is expressed in seconds.
+
+#### SEMAPHORE\_PIPELINE\_QUEUEING\_DURATION
+
+The value of the `SEMAPHORE_PIPELINE_QUEUEING_DURATION` contains the time
+the pipeline spent in the queue.
+
+The value is expressed in seconds.
+
+#### SEMAPHORE\_PIPELINE\_RUNNING\_DURATION
+
+The value of the `SEMAPHORE_PIPELINE_RUNNING_DURATION` contains the pipeline
+execution time while the jobs were running.
+
+The value is expressed in seconds.
+
+#### SEMAPHORE\_PIPELINE\_CREATED\_AT
+
+The value of the `SEMAPHORE_PIPELINE_CREATED_AT` is the UNIX epoch timestamp when
+the pipeline was created.
```

Comment:
An example of the value here and for the other timestamp env vars could be helpful.
We have example values for all other env vars on this page, so it might be good from a consistency standpoint also.